lib/src/widgets/board/entry.dart

```diff
@@ -35,67 +33,71 @@ class Entry extends StatelessWidget {
               overflow: TextOverflow.ellipsis,
               style: const TextStyle(fontSize: AppFontSize.bodyText1),
             ),
-            Icon(
+            const Icon(
               Icons.arrow_drop_up_sharp,
               color: AppColors.accent,
-              size: iconSize,
+              size: _iconSize,
             ),
           ],
         ),
         const SizedBox(width: AppSpacing.xl),
-        Container(
-          decoration: BoxDecoration(
-            color: AppColors.primary,
-            borderRadius: BorderRadius.all(
-              Radius.circular(radius),
+        Expanded(
+          child: Container(
+            decoration: const BoxDecoration(
+              color: AppColors.primary,
+              borderRadius: BorderRadius.all(
+                Radius.circular(_radius),
+              ),
             ),
-          ),
-          child: Row(
-            children: <Widget>[
-              ClipRRect(
-                borderRadius: BorderRadius.all(
-                  Radius.circular(radius),
-                ),
-                child: SizedBox(
-                  height: imageSize,
-                  width: imageSize,
-                  child: Image.asset(
-                    'assets/mock/pictures/$picture',
-                    fit: BoxFit.cover,
+            child: Row(
+              children: <Widget>[
+                ClipRRect(
```

Comment:
cool, allerdings brauchst du das gar nicht. Nimm Container statt der sized box, in der Dekoration kannst du shape auf BoxShape.circle setzen. im Container gibts clip und da nimmst du dann Clip.antiAlias